### ISSUE TITLE
Change criterion attribute `mse` to `squared_error` of the RandomForestRegressor

### ISSUE THREAD

**Issue by aaguasca** [cta-observatory/cta-lstchain#865]
I have trained RFs with the script lstchain_mc_trainpipe.py from the lstchain v0.8.4. Everything goes OK, but I obtain the following warning:

```
/fefs/aswg/workspace/arnau.aguasca/anaconda3/envs/my-lst-dev/lib/python3.8/site-packages/sklearn/ensemble/_forest.py:396: FutureWarning: Criterion 'mse' was deprecated in v1.0 and will be removed in version 1.2. Use `criterion='squared_error'` which is equivalent.
  warn(
/fefs/aswg/workspace/arnau.aguasca/anaconda3/envs/my-lst-dev/lib/python3.8/site-packages/sklearn/ensemble/_forest.py:396: FutureWarning: Criterion 'mse' was deprecated in v1.0 and will be removed in version 1.2. Use `criterion='squared_error'` which is equivalent.
  warn(
/fefs/aswg/workspace/arnau.aguasca/anaconda3/envs/my-lst-dev/lib/python3.8/site-packages/sklearn/ensemble/_forest.py:396: FutureWarning: Criterion 'mse' was deprecated in v1.0 and will be removed in version 1.2. Use `criterion='squared_error'` which is equivalent.
  warn(
/fefs/aswg/workspace/arnau.aguasca/anaconda3/envs/my-lst-dev/lib/python3.8/site-packages/sklearn/ensemble/_forest.py:396: FutureWarning: Criterion 'mse' was deprecated in v1.0 and will be removed in version 1.2. Use `criterion='squared_error'` which is equivalent.
  warn(
```
Can I change the criterion attribute `mse` to `squared_error` in the config files to avoid future warnings?

**Comment by maxnoe** [comment]
Yes, when doing that please add a requirement to `scikit-learn~=1.0` in setup.py `scikit-learn=1` in environment.yml

**Comment by maxnoe** [comment]
Fixed in #867
